### PR TITLE
Get client's address on server side

### PR DESCRIPTION
**Pull request by overvenus**
We can log client's adress after this PR.

CC @AndreMouche

### CODE REVIEW

File: tests/cases/mod.rs [hunk 1]
```diff
@@ -14,3 +14,4 @@
 mod alarm;
 mod health_check;
 mod metadata;
+mod etc;
```
[the remark this answers is not in the document]
Comment (siddontang) [reply tests/cases/mod.rs]: util? or misc?

**Comment by BusyJay** [comment]
LGTM

File: src/call/server.rs [hunk 2]
```diff
@@ -117,14 +118,11 @@ impl RequestContext {
         self.ctx
     }
 
-    fn take_call(&mut self, cq: CompletionQueue) -> Option<Call> {
+    fn call(&mut self, cq: CompletionQueue) -> Call {
```
Comment (hicqu) [inline src/call/server.rs]: Could `Call` lives longer than `RequestContext`? What will happen if call `grpc_call_unref` after `grpc_sys::grpcwrap_request_call_context_destroy`?

**Comment by hicqu** [comment]
rest LGTM.

File: src/call/server.rs (hunk 2 above)
Comment (overvenus) [reply src/call/server.rs]: > Could Call lives longer than RequestContext? 

Yes.

> What will happen if call grpc_call_unref after grpc_sys::grpcwrap_request_call_context_destroy?

Decreases ref counter and cancels the call if needed.

**Pull request merged**